frontend/app/src/services/statusService.js

```diff
@@ -8,13 +8,13 @@ export default {
   async serviceGetStatusById(id) {
     return appAxios().get(`status/view/${id}`);
   },
-  async serviceAddStatus(data) {
-    return appAxios().post(`status/add`, data);
-  },
-  async serviceUpdateStatus(id, data) {
-    return appAxios().put(`/status/update/${id}`, data);
-  },
-  async serviceDeleteStatusById(id) {
-    return appAxios().delete(`status/delete/${id}`);
-  },
+  // async serviceAddStatus(data) {
```

Comment:
it is best practice to remove the commented code. we can always retrieve old code from earlier commits